extensions/objects/templates/image_with_regions_editor.html

```diff
@@ -13,7 +13,8 @@
     ng-mousedown="onSvgMouseDown($event)"
     ng-mousemove="onSvgMouseMove($event)"
     ng-mouseup="onSvgMouseUp($event)">
-    <image xlink:href="<[getPreviewUrl($parent.$parent.value.imagePath)]>"
+    <image ng-href="<[getPreviewUrl($parent.$parent.value.imagePath)]>"
+      xlink:href = ""
```

Comment:
For consistency, let's leave out the spaces around the `=`. I would also recommend add a comment explaining why this is necessary.